litestream.yaml

```diff
@@ -0,0 +1,13 @@
+dbs:
```

Comment:
Checkout how our Envoy container renders a template on startup: https://github.com/ar-io/ar-io-node/blob/develop/envoy/docker-entrypoint.sh#L12

Reply:
it's been adjusted to follow the envoy pattern

Reply:
no what am I saying :D I need to first push my local changes 🤦